models/api.articles.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.fetchArticles = () => {
+  const sqlQuery = `SELECT articles.article_id, articles.title, articles.author, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
+       COUNT(comments.comment_id) AS comment_count 
+FROM articles
+LEFT JOIN comments 
+ON comments.article_id = articles.article_id
+GROUP BY articles.article_id
+ORDER BY articles.created_at DESC; `;
+
```

Comment:
great work with this model! 

you could have a look into casting - and see whether you can change the data type of comment_count